pkg/docker/docker.go

```diff
@@ -308,7 +308,7 @@ func New(config *api.DockerConfig, auth api.AuthConfig) (Docker, error) {
 	if err != nil {
 		return nil, err
 	}
-	k8sDocker := dockertools.ConnectToDockerOrDie(config.Endpoint, 0)
```

Comment:
yeah, again, the backing off of k8s minimally has some bearing, but not passing any endpoint and always relying on the ENV vars does not seem correct.